components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1279,6 +1312,11 @@ where
                 self.register_raft_gc_log_tick();
                 self.register_check_leader_lease_tick();
             }
+
+            if self.fsm.peer.force_leader && r == StateRole::Follower {
+                // for some reason, it's not leader anymore
+                warn!("step to follower in force leader state");
```

Comment:
How could this happen? (And if we need this log, `peer_id` and `region_id` should be included.)